packages/client/CHANGELOG.md

```diff
@@ -6,6 +6,95 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 (modification: no type change headlines) and this project adheres to
 [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## 0.9.0 - 2023-10-26
+
+This client release now syncs with the new Holesky and Dencun devnet-10 test networks and comes with improved sync performance, a revamped post-Merge client UX experience and various beacon sync related fixes and robustness improvements.
+
+### Holesky Testnet Support
+
+This client release now fully supports running the new [Holesky](https://holesky.ethpandaops.io/) public Ethereum testnet replacing the `Goerli` test network, see , PR [#2982](https://github.com/ethereumjs/ethereumjs-monorepo/pull/2982) and some following PRs.
+
+The following command starts an EthereumJS client on Holesky:
+
+```typescript
+ethereumjs --network=holesky --rpc --rpcEngine
+```
+
+Then start a corresponding CL client (e.g. Lodestar with checkpoint sync towards `https://lodestar-holesky.chainsafe.io`), also see client [README](https://github.com/ethereumjs/ethereumjs-monorepo/tree/master/packages/client/README.md).
+
+### Dencun devnet-10 Compatibility
+
+Another testnet to be run with this client: Dencun `devnet-10`, which is one of the last and eventually the _very_ last testnet before running the Dencun hardfork on the official testnet.
```

Comment:
we already a long running devnet-11 now https://forkmon.dencun-devnet-11.ethpandaops.io (3-4 weeks long)  and also since there is again an iteration on the structure of blobsidecars (https://github.com/ethereum/consensus-specs/pull/3531), this won't be the last (although all of that is CL side)